big_tests/tests/push_http_SUITE.erl

```diff
@@ -156,7 +148,7 @@ push_to_many(Config) ->
 start_pool() ->
     Pool = {http, host, http_pool,
             [{strategy, random_worker}, {call_timeout, 5000}, {workers, 20}],
-            [{path_prefix, "/"}, {http_opts, []}, {server, http_notifications_host()}]},
+            [{path_prefix, "/"}, {http_opts, #{}}, {server, http_notifications_host()}]},
```

Comment:
To be precise, these `http_opts` have nothing to do with gun's `http_opts` configuration. This naming is a coincidence, and it would maybe better be named simply `opts`. This map is simply what is given to `gun:open/2`, and one of its parameters could be a `http_opts` with http specific configurations (https://ninenines.eu/docs/en/gun/1.3/manual/gun/)

Reply:
Yes, this is true. These are just the options passed to `mongoose_gun_worker` and then they are used when establishing the connection. The name is a leftover, but for example "ConnOpts" are used as the name in `mongoose_wpool_http` for the broader set of options (the server, path prefix, request timeout and also the "http_opts").